From b185fa632de34032999b5f9b351a1050c93c1b12 Mon Sep 17 00:00:00 2001
From: Iain Sandoe <iain@sandoe.co.uk>
Date: Wed, 9 Sep 2020 19:55:06 +0100
Subject: [PATCH] testsuite, Darwin, Arm64 : Disable the AAPCS64 tests.

These are not appropriate for darwinpcs / Mach-O.

For now, we  have compat.exp and struct-layout-1.exp as some
coverage, but ideally we should have some specific ones (perhaps
some of the AAPCS64 ones can be adapted).

(cherry picked from commit 2d107dadd4bd8c4cadb24bb13722446168384e76)
Signed-off-by: Kirill A. Korinsky <kirill@korins.ky>
---
 gcc/testsuite/gcc.target/aarch64/aapcs64/aapcs64.exp | 5 +++++
 1 file changed, 5 insertions(+)

diff --git gcc/testsuite/gcc.target/aarch64/aapcs64/aapcs64.exp gcc/testsuite/gcc.target/aarch64/aapcs64/aapcs64.exp
index fc8de933d2c..a86e7f340f7 100644
--- gcc/testsuite/gcc.target/aarch64/aapcs64/aapcs64.exp
+++ gcc/testsuite/gcc.target/aarch64/aapcs64/aapcs64.exp
@@ -25,6 +25,11 @@ if { ![istarget aarch64*-*-*] } then {
   return
 }
 
+if { [istarget *-*-darwin*] } then {
+  # darwinpcs and mach-o will need different test mechanisms.
+  return
+}
+
 torture-init
 set-torture-options $C_TORTURE_OPTIONS
 set additional_flags "-W -Wall -Wno-abi"
-- 
2.40.1

